src/openrct2/scripting/ScTile.hpp

```diff
@@ -446,6 +501,114 @@ namespace OpenRCT2::Scripting
             }
         }
 
+        uint16_t mazeEntry_get() const
+        {
+            auto el = _element->AsTrack();
+            if (el != nullptr)
```

Comment:
Should we have an extra check to make sure this is a maze entry?

Reply:
Probably. Consequently, we should then also check in `sequence_get` that it is _not_ a maze. Also:
`Footpath->AdditionStatus` vs `Footpath->rideIndex`
`Track->ColourScheme` (i.e. `colourScheme` and `seatRotation`) vs `Track->MazeEntry`
`Track->BrakeBoosterSpeed` vs it's not a break/booster.
That's all unions used.

Reply:
@IntelOrca what are your opinions

Reply:
I would say so.